addons/mail/static/src/models/res_users_settings.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, many, one } from '@mail/model/model_field';
+
+/**
+ * Mirrors the fields of the python model res.users.settings.
+ */
+registerModel({
+    name: 'res.users.settings',
+    identifyingFields: ['id'],
+    fields: {
+        id: attr({
+            readonly: true,
+            required: true,
+        }),
+        is_discuss_sidebar_category_channel_open: attr(),
+        is_discuss_sidebar_category_chat_open: attr(),
+        push_to_talk_key: attr(),
+        use_push_to_talk: attr(),
+        user_id: one('User', {
```

Comment:
I would expect this to be `res.users`, a python model.

Reply:
Let's not introduce all of those models now, it's probably going to be more trouble than it's worth.